plugin/telescope.lua

```diff
@@ -0,0 +1,144 @@
+if 1 ~= vim.fn.has "nvim-0.7.0" then
+  vim.api.nvim_err_writeln "Telescope.nvim requires at least nvim-0.7.0. See `:h telescope.changelog-1851`"
+  return
+end
+
+if vim.g.loaded_telescope then
+  return
+end
+vim.g.loaded_telescope = true
```

Comment:
```suggestion
vim.g.loaded_telescope = 1
```
for vimscript compatibility

Reply:
makes sense

Reply:
oh also have to switch to == 1 then

Reply:
Ah, I thought you just wanted to test for existence, basically. But yes, to be on the safe side, you should absolutely test for equality with `1`!